Reorganização da hierarquia de pastas de Assets

jpholanda opened the pull request:
Hierarquia de Assets refinada. Images, Prefabs e Scripts tem seus arquivos classificados em pastas de acordo com a entidade do jogo a qual se referem. Scripts são separados de forma mais detalhada. 
Renomeado script "BaseMovement" para "EnemyGridMovement".

Commit:
Reorganização da hierarquia de pastas de Assets

Hierarquia de Assets refinada. Images, Prefabs e Scripts tem seus arquivos classificados em pastas de acordo com a entidade do jogo a qual se referem. Scripts são separados de forma mais detalhada. 
Renomeado script "BaseMovement" para "EnemyGridMovement".

giordanorn reviewed (approved):
Top. @jpholanda  pode se encarregar de realizar o merge hehe.

Commit:
Adicionado pasta Tiles de volta em Images

Tiles estava vazia, então não entrou no commit anterior. Adicionei um arquivo .keep para mantê-la existindo mesmo sem nada dentro.

Commit:
Merge branch 'master' into ajustes-e-organização
